airbyte-integrations/template/singer-source/README.md

```diff
@@ -0,0 +1 @@
+# This is a demo source
```

Comment:
We'll probably want a doc outline here at some point.

Reply:
see singer PR